content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,280 @@ weight: 50
+6.  Install `cluster.env` under `/var/lib/istio/envoy`.
 
     {{< text bash >}}
-    $ host istio-pilot.istio-system.svc.cluster.local.
-    {{< /text >}}
 
-    Example generated message:
+    $ sudo cp cluster.env /var/lib/istio/envoy
 
-    {{< text plain >}}
-    istio-pilot.istio-system.svc.cluster.local has address 10.63.247.248
     {{< /text >}}
 
-    Check istio-ingress similarly:
+5.  Transfer ownership of the files in `/etc/certs/` and `/var/lib/istio/envoy` to the Istio proxy.
 
     {{< text bash >}}
-    $ host istio-ingress.istio-system.svc.cluster.local.
-    {{< /text >}}
-
-    Example generated message:
 
-    {{< text plain >}}
-    istio-ingress.istio-system.svc.cluster.local has address 10.63.243.30
-    {{< /text >}}
+    $ sudo chown -R istio-proxy /etc/certs /var/lib/istio/envoy
 
-*   Verify connectivity by checking whether the VM can connect to Pilot and to an endpoint.
-
-    {{< text bash json >}}
-    $ curl 'http://istio-pilot.istio-system:8080/v1/registration/istio-pilot.istio-system.svc.cluster.local|http-discovery'
-    {
-      "hosts": [
-       {
-        "ip_address": "10.60.1.4",
-        "port": 8080
-       }
-      ]
-    }
     {{< /text >}}
 
-    On the VM, use the address above. It will directly connect to the pod running istio-pilot.
+7.  Verify the node agent works:
 
     {{< text bash >}}
-    $ curl 'http://10.60.1.4:8080/v1/registration/istio-pilot.istio-system.svc.cluster.local|http-discovery'
-    {{< /text >}}
 
-*   Extract the initial Istio authentication secrets and copy them to the machine. The default
-installation of Istio includes Citadel and will generate Istio secrets even if
-the automatic `mTLS`
-setting is disabled (it creates a secret for each service account, and the secret
-is named as `istio.<serviceaccount>`). It is recommended that you perform this
-step to make it easy to enable mutual TLS in the future and to upgrade to a future version
-that will have mutual TLS enabled by default.
+    $ sudo node_agent
+    ....
+    CSR is approved successfully. Will renew cert in 1079h59m59.84568493s
 
-    `ACCOUNT` defaults to 'default', or `SERVICE_ACCOUNT` environment variable
-    `NAMESPACE` defaults to current namespace, or `SERVICE_NAMESPACE` environment variable
-    (this step is done by machineSetup)
-    On a Mac either `brew install base64` or `set BASE64_DECODE="/usr/bin/base64 -D"`
+    {{< /text >}}
+
+8.  Start Istio using `systemctl`.
 
     {{< text bash >}}
-    $ @install/tools/setupMeshEx.sh@ machineCerts ACCOUNT NAMESPACE
+
+    $ sudo systemctl start istio-auth-node-agent
+    $ sudo systemctl start istio
+
     {{< /text >}}
 
-    The generated files (`key.pem`, `root-cert.pem`, `cert-chain.pem`) must be copied to /etc/certs on each machine, readable by istio-proxy.
+After setup, the machine can access services running in the Kubernetes cluster
+or other mesh expansion machines.
```

Comment:
Preposition missing?
```suggestion
or in other mesh expansion machines.
```

Reply:
The "in" covers both of them, though rereading it's probably better to say *on* the mesh expansion machines.